source/extensions/omni.isaac.lab/omni/isaac/lab/utils/math.py

```diff
@@ -988,7 +988,10 @@ def transform_points(
 
 @torch.jit.script
 def unproject_depth(depth: torch.Tensor, intrinsics: torch.Tensor) -> torch.Tensor:
-    r"""Unproject depth image into a pointcloud.
+    r"""Unproject depth image into a pointcloud. If the depth image originates
+    from the distance_to_camera replicator, is recommended to call
+    convert_perspective_depth_image_to_orthogonal_depth_image on your depth
```

Comment:
Using these directives help link in the docs :)

More info: https://www.sphinx-doc.org/en/master/usage/domains/python.html#cross-referencing-python-objects

```suggestion
    :meth:`convert_perspective_depth_image_to_orthogonal_depth_image` on your depth
```